SupportClient/SupportClient/Controllers/ViewController.swift

```diff
@@ -47,10 +48,10 @@ class ViewController: UIViewController {
     }
     
     func setupNavigationBar() {
-        navigationController?.navigationBar.barTintColor = .navigationTintColor
+        navigationController?.navigationBar.barTintColor = ._navigationTintColor
```

Comment:
We were already doing this to the fonts already, but adding the `_` makes the autocomplete so much faster, because then you can just type `._` to see all your custom fonts instead of trying to remember what you named them. Probably not convention, but is easier